fs/config/config.go

```diff
@@ -214,8 +214,17 @@ var errorConfigFileNotFound = errors.New("config file not found")
 // automatically decrypt it.
 func loadConfigFile() (*goconfig.ConfigFile, error) {
 	b, err := ioutil.ReadFile(ConfigPath)
+	envpw := os.Getenv("RCLONE_CONFIG_PASS")
 	if err != nil {
 		if os.IsNotExist(err) {
+			if len(configKey) == 0 && envpw != "" {
```

Comment:
I think you should move this block (and the `envpw := os.Getenv("RCLONE_CONFIG_PASS")`) right to the start of the function, then we can remove the duplicate code below